### PR TITLE
4.7.0

### PR DESCRIPTION
**Pull request by pdion891**
Please review what's new section and approuve or not the use of a screenshot for the Metric section.

Thanks,

### CODE REVIEW

**Comment by runseb** [comment]
Looks good, but I think it's missing some features:
- quota
- Nuage ?

pinging @remibergsma

**Comment by remibergsma** [comment]
- Quota plugin, see PR 768 for screenshots.
- Nicira and Nuage plugins updated
- UI improvements
- >100 smaller bugfixes and improvements

File: source/upgrade/_sysvm_templates_pre45.rst [hunk 1]
```diff
@@ -18,8 +18,9 @@
 Update System-VM templates
 --------------------------
 
-.. note::
-   Upgrading pre-4.3 to 4.4.1 require 2 systemvm templates downloaded: the 4.3 and 4.4.
+.. warning::
+   Upgrading from 4.4 or older to 4.6.0 require 2 systemvm templates to be
```
Comment (remibergsma) [inline source/upgrade/_sysvm_templates_pre45.rst]: You can remove this line, you just need the 4.6 version (as there is no change in 4.7)

Comment (pdion891) [inline source/upgrade/_sysvm_templates_pre45.rst]: This file ``_sysvm_templates_pre45.rst`` is not used in the RN, I left it there as reference.

**Pull request closed without merging**